src/sprite_actor.cpp

```diff
@@ -80,7 +80,7 @@ void Sprite_Actor::Update() {
 			// Is a battle charset animation
 
 			static const int frames[] = {0,1,2,1,0};
-			int frame = (battler->IsDefending() ? 0 : frames[cycle / 10]);
+			int frame = (battler->IsDefending() ? 0 : (normal_attacking ? std::min(2, cycle / 10) : frames[cycle / 10]));
```

Comment:
Can you explain what normal_attacking is?

Reply:
normal_attacking is set to true if the actor starts a normal attack. If this flag is set, the frame sequence is only played once and stays at the last frame until the actor finishes his attack.